src/views/Player.vue

```diff
@@ -186,6 +190,22 @@ export default {
 
           // set the global list of time values
           this.times = res.data.times
+
+          // merge the previous session data
+          if (res.data.sessionData != '') {
+            this.answers = res.data.sessionData.answers
+            for (i=0; i < questions.length; i++){
```

Comment:
I think this loop will look cleaner with a `question.forEach(question, index) { }` loop